client/api/python/heketi/heketi.py

```diff
@@ -55,7 +55,7 @@ def _set_token_in_header(self, method, uri, headers={}):
         claims['qsh'] = hashlib.sha256(val).hexdigest()
 
         token = jwt.encode(claims, self.key, algorithm='HS256')
-        headers['Authorization'] = b'bearer ' + token
+        headers['Authorization'] = 'bearer ' + token
```

Comment:
Why is changing the type from bytes to (unicode) str the right approach? Why should token be encoded to bytes instead? Also if this recently changed perhaps it should handling token being either str or bytes?

Reply:
Actually, I was thinking that we are getting str from jwt.encode() and did not want to touch it. Moreover, we are creating bytes ourselves that's the main reason to change it this way.

That's also possible that jwt.encode() might be sending bytes before instead of string.